scripts/trim-low-abund.py

```diff
@@ -488,6 +520,28 @@ def main():
         log_info("Saving k-mer countgraph to {graph}", graph=args.savegraph)
         ct.save(args.savegraph)
 
+    if args.summary_info is not None:
+        # when the output is stdout the name is == 1
+        if args.output is not None and args.output.name != 1:
```

Comment:
Reworded it, what I am trying to do is leave a hint as to why `1` is a special name/value. Basically trying to explain the weird condition of the if which is "if we have an explicit name for the output use it unless that name is 1 which is the magic value of stdout"